pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/vedlegg/maanedligPensjonFoerSkatt/Grunnpensjon.kt

```diff
@@ -0,0 +1,471 @@
+    val beregnetPensjonPerManedGjeldende: Expression<MaanedligPensjonFoerSkattDto.AlderspensjonPerManed>,
+    val epsGjeldende: Expression<MaanedligPensjonFoerSkattDto.EPSgjeldende?>,
+    val alderspensjonGjeldende: Expression<MaanedligPensjonFoerSkattDto.AlderspensjonGjeldende>,
+    val institusjonsoppholdGjeldende: Expression<MaanedligPensjonFoerSkattDto.InstitusjonsoppholdGjeldende?>,
+    val bruker: Expression<MaanedligPensjonFoerSkattDto.Bruker>,
+    val beregnetSomEnsligPgaInstopphold: Expression<Boolean>,
+): OutlinePhrase<LangBokmalNynorskEnglish>() {
+    override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+        val epsBorSammenMedBruker = epsGjeldende.borSammenMedBruker_safe.ifNull(false)
+        val epsMottarPensjon = epsGjeldende.mottarPensjon_safe.ifNull(false)
+        val epsHarInntektOver2G = epsGjeldende.harInntektOver2G_safe.ifNull(false)
+
+        val beregnetEtterEgen =
+            beregnetPensjonPerManedGjeldende.beregnetEtter_safe.ifNull(AlderspensjonBeregnetEtter.AVDOD)
+                .isOneOf(AlderspensjonBeregnetEtter.EGEN)
+        val beregnetEtterAvdod =
+            beregnetPensjonPerManedGjeldende.beregnetEtter_safe.ifNull(AlderspensjonBeregnetEtter.EGEN)
+                .isOneOf(AlderspensjonBeregnetEtter.AVDOD)
+
+        val grunnbeloep = beregnetPensjonPerManedGjeldende.grunnbeloep.format()
+
+        val grunnpensjonSats = alderspensjonGjeldende.grunnpensjonSats.format()
+        ifNotNull(
+            beregnetPensjonPerManedGjeldende.grunnpensjon
+        ) { grunnpensjon ->
+            showIf(beregnetPensjonPerManedGjeldende.fullTrygdetid and beregnetEtterEgen) {
+                showIf(alderspensjonGjeldende.regelverkstype.isOneOf(AP1967, AP2011, AP2016)) {
+                    paragraph {
+                        text(
+                            Bokmal to "Grunnpensjon ",
+                            Nynorsk to "Grunnpensjon ",
+                            English to "The basic pension ",
+                            FontType.BOLD
+                        )
+                        textExpr(
+                            Bokmal to "fastsettes med utgangspunkt i folketrygdens grunnbeløp, som for tiden er ".expr() + grunnbeloep + " kroner.",
+                            Nynorsk to "blir fastsett med utgangspunkt i grunnbeløpet i folketrygda, som for tida er ".expr() + grunnbeloep + " kroner.",
+                            English to "is calculated on the basis of the National Insurance basic amount (G), which is currently NOK ".expr() + grunnbeloep + ".",
+                        )
+                    }
+                }
+
+
+                showIf(
+                    bruker.sivilstand.isOneOf(
+                        ENSLIG,
+                        ENKE,
+                        GLAD_EKT,
+                        GLAD_PART,
+                        SEPARERT,
+                        SEPARERT_PARTNER,
+                        GIFT,
```

Comment:
gift og partner bør vel ikkje vera her?

Reply:
Ikke? Det gir mening for meg. Om du er gift/partner og beregnet som enslig på grunn av institusjonsopphold så får de beskjed om at de er enslig pensjonist. Ellers er du ikke det og frasen vises.

Reply:
Jo, den er eg med på, men på linje 103 under har du ein or med GIFT, PARTNER and beregnetSomEnslig, da bør dei ikkje også stå her på linje 100 og 101

Reply:
.. for sånn som det står no, vil du bli rekna som einsleg viss du har sivilstatus gift eller partner - uavhengig av om du har institusjonsopphald eller ikkje 🙈

Reply:
Kanskje en parantes som er uklar her, men det ser riktig ut.

Reply:
Det er showIf(A or(B and C))